.gitignore

```diff
@@ -0,0 +1,3 @@
+target/
+.idea
+*.iml
```

Comment:
I'd go for a more [sophisticated](https://github.com/dkowis/finatra-typesafe-config/blob/master/.gitignore) .gitignore configuration.